src/ui.js

```diff
@@ -27,9 +27,7 @@ let ui = {
 let updateGyro = (key, value) => {
     ui.gyro.val = value;
     ui.gyro.visualVal = Math.floor(ui.gyro.val - ui.gyro.offset);
-    if (ui.gyro.visualVal < 0) {
-        ui.gyro.visualVal += 360;
-    }
+    ui.gyro.visualval %= 360
```

Comment:
This is definitely an issue, but this implementation won't work. We need to still have the lines we had before, since modulating won't fix negative values. Modulate first, then add if under 0.

Also, `visualVal` is capitalized incorrectly and you forgot a semicolon.